src/util/scriptQueryParameters.ts

```diff
@@ -1,3 +1,5 @@
+import { QueryParameters } from "../@types";
+
 const getElements = (element: string): Element[] =>
   [].slice.call(document.getElementsByTagName(element));
```

Comment:
Another nit-picky suggestion for readability:
```suggestion
  [...document.getElementsByTagName(element)];
```

Reply:
This function was previous used outside of the feature detection and thus needed to be bullet proof. While I agree the typescript compiler will handle this for us, if you compare the ES% output results for the two options in the playground you'll see why `slice.call()` is still preferred :)